jQuery: unit testing (#3618)
DevExpress/devextreme-documentation: concepts/58 jQuery Components/30 Components Testing/10 Unit Testing.md

> Unit testing implies separate testing of single features (units of the code). A unit can be a function or class. 

Reviewer: ```suggestion
Unit testing allows you to test single features (units of the code). A unit can be a function or class. 
```
